### PR TITLE
fix: call search version on search moment

### PR DESCRIPTION
**Pull request by rebelchris**
### Describe what this PR does
- Need to verify on preview deployment 

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### Preview domain
https://fix-search-version.preview.app.daily.dev

### CODE REVIEW

File: packages/shared/src/components/MainFeedLayout.tsx [hunk 1]
```diff
@@ -218,6 +215,7 @@ export default function MainFeedLayout({
     }
 
     if (isSearchOn && searchQuery) {
+      const searchVersion = getFeatureValue(feature.searchVersion);
```
Comment (nensidosari) [inline packages/shared/src/components/MainFeedLayout.tsx:218]: Just curiosity why are only getting the version here? or we only need it here?

Comment (rebelchris) [reply packages/shared/src/components/MainFeedLayout.tsx:218]: Yeah only needed here,
Was a edge case where the other hook would still be loading and return the default value and then never retrigger.

**Review by nensidosari — APPROVED** [review]

**Review by idoshamun — COMMENTED** [review]
Can we remove the other evaluations of this feature outside this query?

**Comment by rebelchris** [comment]
@idoshamun not sure what you mean, the other ones are different queries (there's only useConditionalFeature left in this regard)
But it's the suggestions and haven't seen that one fail getting the right version.

Not sure if that's what you meant.

**Review by idoshamun — APPROVED** [review]
I thought there was an additional one. But if not we're good to merge!

**Pull request merged**